.github/issue-labeler.yml

```diff
@@ -2,7 +2,7 @@
     - '(visual bug)'
 
 'B: Unofficial Download':
-    - '(AUR|Chocolatey|FreeTubeCordova|PortableApps|winget|Scoop|Snapcraft|MPR|Nix)'
+    - '(AUR \(Unofficial\)|Chocolatey \(Unofficial\)|\.apk \(Android, FreeTubeCordova Unofficial\)|PortableApps \(Unofficial\)|winget \(Unofficial\)|Scoop \(Unofficial\)|Snapcraft \(Unofficial\)|MPR \(Unofficial\)|Nix \(Unofficial\))'
```

Comment:
Is there a need to include `(Unofficial)` here? (Is this exact match not regex?)

Reply:
I included this because in the past i have seen some people mentioning one of the install methods in the issue body but they had not selected it as install method